cordapp/README.md

```diff
@@ -1,13 +1,24 @@
 # CorDapp Sample - Java
 
-This is a simple CorDapp using the Conclave API. It is licensed under the Apache 2 license, and therefore you 
-may copy/paste it to act as the basis of your own commercial or open source apps.
+This is a simple [CorDapp](https://docs.r3.com/en/platform/corda/4.8/open-source/cordapp-overview.html) using the Conclave API. It is licensed under the Apache 2 license. So, you 
+can copy/paste it to act as the basis of your commercial or open source apps.
 
-## Usage
+This CorDapp sample builds on the [hello-world](writing-hello-world.md) sample. It reverses a string using two [nodes](https://docs.r3.com/en/platform/corda/4.8/enterprise/node/component-topology.html). One of these nodes is used to load the enclave.
```

Comment:
You point to writing-hello-world.md document, but that one is not present in this repo. Maybe we can point directly to our docsite (https://docs.conclave.net/writing-hello-world.html)?

Reply:
Corrected.